examples/metrics/README.md

```diff
@@ -0,0 +1,42 @@
+### Running the example to export metrics to Google Cloud
```

Comment:
This doesn't seem related to the fix -- please move everything related to the new example to a separate PR. Also, if we need two different entry points then we should have two different projects.